assets/activation-scripts/activate.d/attach-inplace.bash

```diff
@@ -0,0 +1,45 @@
+# 3. "in-place" mode: emit activation commands in correct shell dialect
```

Comment:
Nit: some of these comments could be fixed up since they still refer to an ordering between activation methods.

Reply:
Done 👍🏻